lib/transformer.js

```diff
@@ -27,6 +27,15 @@ var Transformer = function(schemas, options) {
   this.options = options;
   this.formatter = options.formatter || JSONformatter;
   this.schemas = schemas;
+  var schemaPages = {};
+  _.forEach(this.options.pages, function(page, pageIndex) {
+    _.forEach(page.sections, function(section) {
+      _.forEach(section.schemas, function(schema) {
+        schemaPages[schema] = pageIndex;
```

Comment:
No. It's page specific, not section specific. There could be a problem if you had the same schema referenced in two different pages. This would just use the last defaults for such schema. However, to solve this properly in a more general way, I believe we would have to rewrite big parts of this library. At that point, we want to support usage of the same schema in different pages (or sections) with different transformations.